db/dcrsqlite/apisource.go

```diff
@@ -1242,6 +1242,15 @@ func (db *wiredDB) GetExplorerAddress(address string, count, offset int64) *expl
 		return nil
 	}
 
+	isDummyAddress := IsZeroHashP2PHKAddress(address, db.params)
+	if isDummyAddress {
+		return &explorer.AddressInfo{
+			Address:        address,
+			IsDummyAddress: isDummyAddress,
+			Fullmode:       true,
```

Comment:
also, `Balance:        new(explorer.AddressBalance),` to avoid a potential nil pointer dereference panic in template execution

Reply:
and `UnconfirmedTxns: new(AddressTransactions),` please